addon/components/full-calendar.js

```diff
@@ -19,9 +19,19 @@ export default Ember.Component.extend({
   fixedWeekCount        : true,
   weekNumbers           : false,
   height                : 'auto',
+  businessHours         : { start: '09:00', end: '17:00', dow: [0,1,2,3,4,5,6]},
```

Comment:
The [default](http://fullcalendar.io/docs/display/businessHours/) seems to `false`. Can we use that instead?